index.rst

```diff
@@ -1,51 +1,212 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+.. Add content below. Do not include the document title.
 
-  Sections
-  ========
+.. _Overview:
 
-  and
+Overview
+========
 
-  Subsections
-  -----------
+Ultimately, LSST Alert Production (AP) will need to perform all steps of the Level 1
+pipeline automatically on each new raw image from the telescope. Details
+are in the LSST Data Management Science Pipelines Design (LDM-151, [swi17]_) and 
+the LSST Data Products Definition Document (DPDD, [jur16]_). Briefly
+summarized below and in :ref:`Figure 1<fig-nightly-proc>`, the first portion of these steps are:
 
-  and
+1. Instrument Signature Removal (ISR), which must include crosstalk correction, bad pixel masking, bias subtraction, and flat field division,
```

Comment:
Fringes and brighter-fatter aren't implemented yet, and tree rings are mostly an artifact of screwing up bias or flat corrections. But I'll add a mention of linearity!